core/domain/email_manager.py

```diff
@@ -300,10 +298,10 @@ def send_role_notification_email(
     """
     # Editor role email body and email subject templates.
     # pylint: disable=invalid-name
-    editor_role_email_subject_template = (
+    EDITOR_ROLE_EMAIL_SUBJECT_TEMPLATE = (
```

Comment:
Yeah sure thing. I changed it back to lowercase and removed `editor_role` from beginning of variable name.